binding.cc

```diff
@@ -655,7 +655,7 @@ namespace zmq {
       return Nan::ThrowError("Must pass an option");
     if (!info[0]->IsNumber())
       return Nan::ThrowTypeError("Option must be an integer");
-    int64_t option = Nan::To<int64_t>(info[0]).FromJust();
+    int option = Nan::To<int>(info[0]).FromJust();
```

Comment:
Or should [`GetSockOpt`](https://github.com/zeromq/zeromq.js/blob/master/binding.cc#L112) accept `int64_t`?